### PR TITLE
Okta 438752 build provisioning integration singlepage

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** This PR updates https://developer.okta.com/docs/guides/build-provisioning-integration/overview/ to change it from a multi-page guide to a set of related single page guides — in effect, a multiple-page guide created with single pages, not the multi-page feature, which we are trying to get rid of. It also aims to provide more context on the over view page, and gives the material a less fragmented, more grouped structure.  It follows the plan I outlined in https://docs.google.com/document/d/1KRQ9_JCKWcoj5Nk7eC0yO_GdqDxxDtRoAqpICnELfUU/edit#.
- **Is this PR related to a Monolith release?** No
- Redirects tested on staging 12/6/21. Redirects work as expected.

### Resolves:

* [OKTA-438752](https://oktainc.atlassian.net/browse/OKTA-438752)

### CODE REVIEW

**Review by mauricesharp-okta — COMMENTED** [review]
How about changing lines 103 to 105 to make them clearer and more succinct:

```
**Q: How do I integrate a SCIM application residing inside my corporate firewall with Okta?**

Use the Okta agent-based provisioning solution. For more information, see the [On-Premises Provisioning](https://help.okta.com/okta_help.htm?id=ext_OPP_configure) configuration guide.

The [Build a SCIM provisioning integration](/docs/guides/scim-provisioning-integration-overview) instructions target cloud-based applicationsconfiguration guide.
```

@mauricesharp-okta @chrisdmills-okta  - our style for cross-reference wording is: See the [On-Premises Provisioning](https://help.okta.com/okta_help.htm?id=ext_OPP_configure) configuration guide.

no need for "For more information" unless it is unclear - like when the content discusses two items and the cross reference is sending them to more information about one of the two.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/index.md:3]: How about (I'm assuming it's the Okta console but not 100% sure of that):
``Create and configure SCIM integrations, and check the attributes with their corresponding mappings in the Okta console.``

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:3]: Same excerpt and content suggestions as those above:
'Create and configure SCIM integrations, and check the attributes with their corresponding mappings in the Okta console.'

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:10]: This could be shortened a bit:
- There's no need to include "part of the ..." as their already in the guide. It should be clear from context.
- "This guide" is also somewhat redundant as again, they're in the guide.

How about something like:
Learn how to create a SCIM integration, configure the integration's SCIM options, and check the attributes and corresponding mapping using Okta.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:14]: How about replacing this paragraph (NOTE, I didn't know the title of the article... "previous part of the series" could be confusing if the developer gets here directly from a search) :
Make sure your SCIM implementation passes all the Runscope tests before integrating it with Okta. For more information on resting your SCIM implementation, see [TITLE OF THE ARTICLE GOES HERE](/docs/guides/scim-provisioning-integration-prepare/main/#test-your-scim-api)
Before integration your SCIM using Okta

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:18]: How about changing these two paragraphs:
1. Sign up for an [Okta developer account](https://developer.okta.com/signup/).
1. Click on the link in the signup email to open your Admin Console.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:19]: "In the Admin Console" is redundant as the step before tells them they are already there. How about:
1. Select **Applications** > **Applications**.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:21]: How about (see the next comment for a suggestion for the follow on text):
1. Search for either "SCIM 2.0" or "SCIM 1.1" depending on the version of SCIM supported by your server.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:22]: How about:
1. Click **Add** on the desired template application for your integration and finish adding the template application.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:23]: How about ("in their web browser" is redundant as landing pages are understood to be in a web browser) :
1. On the **General Settings** page: set the name of your application, choose if it's hidden from general and mobile users, and choose if users' are automatically logged in from the landing page. Click **Next**.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:24]: How about:
1. Choose the sign on method for your integration on the **Sign-On Options** page. Select either SAML or SWA. For guidance on choosing the access method, see [Applications topic](https://help.okta.com/okta_help.htm?id=ext_Apps_Apps). Click **Done** to create the integration.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:25]: How about ("After the integration is created" seems redundant):
```
1. Click the **Provisioning*** tab, then in the main panel click **Configure API Integration**. Select the **Enable API Integration** check box.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:27]: How about:
```
Configure the credential options based on the previously chosen authentication method:
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:28]: How about:
```
- Basic Auth: Enter the the username and password for the account that handles the create, update, and deprovisioning actions of your SCIM implementation.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:29]: Same as Basic Auth... how about:
```
- HTTP Header: Enter a bearer token for accessing your SCIM implementation.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:30]: Same as Basic Auth... how about:
```
 - OAuth: Enter the OAuth token for accessing your SCIM implementation.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:32]: How about:
```
Test the credentials by clicking **Test API Credentials**, which attempts to connect to Okta. If there's an error, check the credentials entered above.
```

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-overview/index.md
Comment (mauricesharp-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-overview/index.md:3]: How about changing this so that the content is mentioned first (similar comments on "Learn about..."... it's a guide so that's implicit... well, if it's a good guide anyway and ours are ;-)):
```
Create a provisioning integration with SCIM in Okta.
```

File: packages/@okta/vuepress-site/docs/concepts/scim/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/concepts/scim/index.md:12]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/index.md:3]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:3]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:10]: Done, mostly. I like having the text slightly more conversational, so I've kept "this guide"

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:14]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:18]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:21]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:22]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:23]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:24]: done, mostly. We tend to say sign in rather than sign on.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:25]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:27]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:28]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:29]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:30]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md:32]: done

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-overview/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-overview/index.md:3]: done

**Comment by chrisdmills-okta** [comment]
Cheers @mauricesharp-okta ! Feedback implemented.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-overview/main/index.md
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-overview/main/index.md:10]: "guide series" <-- I quite like this. Useful, where appropriate.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/main/index.md
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/main/index.md:10]: See previous comment.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md:26]: Do these "minimum requirements" merit inclusion under **What you need** in the nut facts?

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/main/index.md
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/main/index.md:13]: Does the "Prerequisites" section on the Test page also merit inclusion in the nut facts?

**Review by brentschaus-okta — APPROVED** [review]
LGTM

Just a few comments and queries.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-overview/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-overview/main/index.md:42]: Agreed; fixed in my next commit.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/main/index.md:10]: fixed

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/index.md:6]: fixed!

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-prepare/main/index.md:26]: This is a good point. I'm going to add a link to this place in the overview nut facts.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-test/main/index.md:13]: I think so. Add a link to this to, from the overview nut facts.

**Comment by chrisdmills-okta** [comment]
Thanks for the comments @brentschaus-okta ! I've made all the fixes you suggested. 

I'm going to hold off merging this until @IanHakes-okta has merged his changes in https://github.com/okta/okta-developer-docs/pull/2870, then I'll make sure those are represented in this PR.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md]: OK, thanks @susanharper-okta ; updated in next commit.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md]: updated.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md]: Updated.

File: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md]: There were two instances; I've updated both.

**Pull request merged**
